pyproject.toml

```diff
@@ -38,6 +38,7 @@ python-socketio = "^5.7.2"
 psutil = "^5.9.4"
 websockets = "^10.4"
 cloudpickle = "^2.2.1"
+requests = "^2.28.2"
```

Comment:
I think we can use the `httpx` library that's installed already - should follow the same API as `requests`